charts/karavi-observability/templates/cert-manager.yaml

```diff
@@ -0,0 +1,132 @@
+{{- if and (.Values.karaviTopology.certificateFile) (.Values.karaviTopology.privateKeyFile) }}
+{{- $certificateFileContents := .Files.Get .Values.karaviTopology.certificateFile }}
+{{- $privateKeyFileContents := .Files.Get .Values.karaviTopology.privateKeyFile }}
+apiVersion: v1
+data:
+  tls.crt: {{ $certificateFileContents | b64enc }}
+  tls.key: {{ $privateKeyFileContents | b64enc }}
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: karavi-topology-secret
+  namespace: {{ .Release.Namespace }}
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: karavi-topology-issuer
+  namespace: {{ .Release.Namespace }}
+spec:
+  ca:
+    secretName: karavi-topology-secret
+
+---
+{{- end }}
+
+{{- if and (.Values.otelCollector.certificateFile) (.Values.otelCollector.privateKeyFile) }}
+{{- $certificateFileContents := .Files.Get .Values.otelCollector.certificateFile }}
+{{- $privateKeyFileContents := .Files.Get .Values.otelCollector.privateKeyFile }}
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: otel-collector-secret
+  namespace: {{ .Release.Namespace }}
+data:
+  tls.crt: {{ $certificateFileContents  | b64enc }}
+  tls.key: {{ $privateKeyFileContents | b64enc }}
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: otel-collector-issuer
+  namespace: {{ .Release.Namespace }}
+spec:
+  ca:
+    secretName: otel-collector-secret
+
+---
+{{- end }}
+
+{{- if or (and (not .Values.karaviTopology.certificateFile) (not .Values.karaviTopology.privateKeyFile)) (and (not .Values.otelCollector.certificateFile) (not .Values.otelCollector.privateKeyFile)) }}
```

Comment:
Can you add a comment above this line to make it easier to understand this condition that no certificate or key files have been provided.

Reply:
How does this comment look?

Reply:
Looks good